### PR TITLE
Android/feature/mainpage merge with master

### PR DESCRIPTION
**Pull request by Hazerank**
The main page is implemented according to the design. There are still a few missing points. The connection with backend should be done.

### CODE REVIEW

**Review by fihlie — APPROVED** [review]
Our containers conflicted, we should check that afterwards, otherwise it works great. Let's merge and check

File: source/android/Renaissance/lib/home_page.dart [hunk 1]
```diff
@@ -9,14 +13,238 @@ class HomePage extends StatefulWidget {
 }
 
 class _HomePageState extends State<HomePage> {
+  var _pageIndex = 0;
+
+  void changePage(int num) {
+    setState(() {
+      _pageIndex = num;
+      print("Deneme");
```
Comment (kursatbakis) [inline source/android/Renaissance/lib/home_page.dart:21]: These print lines should be removed

File: source/android/Renaissance/lib/home_page.dart [hunk 2]
```diff
@@ -9,14 +13,238 @@ class HomePage extends StatefulWidget {
...
+      {"name": "Kar", "effort": 2, "rating": 1.6},
+      {"name": "Araba", "effort": 2, "rating": 1.7},
+      {"name": "Selam", "effort": 2, "rating": 1.8},
+      {"name": "Muz", "effort": 2, "rating": 1.8},
+      {"name": "Ahoy", "effort": 2, "rating": 1.9},
+      {"name": "Cam", "effort": 2, "rating": 1.5},
+      {"name": "Kar", "effort": 2, "rating": 1.6},
+      {"name": "Araba", "effort": 2, "rating": 1.7},
+    ];
+    return MaterialApp(
```
Comment (kursatbakis) [inline source/android/Renaissance/lib/home_page.dart:68]: You should not return a new app in a page. You should return Scaffold.

File: source/android/Renaissance/lib/home_page.dart (hunk 1 above)
Comment (Hazerank) [reply source/android/Renaissance/lib/home_page.dart:21]: Okay, the print statements were for debugging.

File: source/android/Renaissance/lib/widget/tag_container.dart [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                children: [
+                  Text(course_name,
+                      overflow: TextOverflow.clip,
+                      maxLines: 2,
+                      textAlign: TextAlign.center,
+                      style: TextStyle(fontSize: 14)),
+                  const SizedBox(height: 5),
+                  MaterialButton(
+                    shape: StadiumBorder(),
+                    color: Color(0x99FFFFFF),
```
Comment (kursatbakis) [inline source/android/Renaissance/lib/widget/tag_container.dart:34]: Use AppColor

**Review by kursatbakis — CHANGES_REQUESTED** [review]
We are going to merge when you do the changes.

File: source/android/Renaissance/lib/home_page.dart (hunk 2 above)
Comment (Hazerank) [reply source/android/Renaissance/lib/home_page.dart:68]: Thanks, It was a rookie mistake while I was learning  the flutter.

File: source/android/Renaissance/lib/widget/tag_container.dart (hunk 3 above)
Comment (Hazerank) [reply source/android/Renaissance/lib/widget/tag_container.dart:34]: It is done and added to the MyColors as lightGrey

**Review by kursatbakis — APPROVED** [review]
Nice

**Pull request merged**